tools/sigh.js

```diff
@@ -581,20 +589,23 @@ function test(args) {
   }
 
   const runner = buildTestRunner();
-  return saneSpawn(
-      'node',
-      [
-        '--experimental-modules',
-        '--trace-warnings',
-        '--no-deprecation',
-        ...extraFlags,
-        '--loader',
-        fixPathForWindows(path.join(__dirname, 'custom-loader.mjs')),
-        '-r',
-        'source-map-support/register.js',
-        runner
-      ],
-      {stdio: 'inherit'});
+  for (var i = 0; i < JSON.stringify(options.repeat || 1); i++) {
```

Comment:
Needs some simple docs.  A comment will do.

Can you hoist the options.repeat value into a constant.  That will make it obvious what's happening,